### ISSUE TITLE
Avoiding keypress events in editable divs

### ISSUE THREAD

**Issue by Nishant-Pall** [arthurtyukayev/use-keyboard-shortcut#49]
Is it possible to stop event triggering if i'm editing an editable div? The triggers work perfectly fine, and don't work in input elements which is expected but trigger if I'm editing an editable div. Is there a way to avoid that? Thanks

**Comment by arthurtyukayev** [comment]
From what I can tell here:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/contentEditable

Making a `div` editable requires just passing the HTML attribute `contentEditable`. I think that it might be a better idea to check for `contentEditable` being truthy in the `KeydownEvent` rather than using the CSS classes approach. 

cc @thmi-inx

**Comment by thmi-inx** [comment]
I guess your approach could work for my use-case. But using a CSS class, gives you more control. You might have two editable HTML elements and only one of them should react to your shortcut and the other one should ignore it ... ?

But i am not sure if there is a real case for this, maybe you would want one editable to trigger an action and add a letter to the editable div, and all other editable divs should not trigger the action when typing ??? 

More realistic might be, that you define, lets say "Cmd + S", to trigger an action for one element and  you want all other other to ignore it. BUT then you should be better off inverting the idea and only allow the specified Class to trigger the event. 

In short, you idea sounds right and would work for me. The other cases i listed seem a bit arbitrary.

**Comment by Nishant-Pall** [comment]
I worked out a way to return if the target name matches any input or textarea

```const handleKeyPress = useCallback(
    event => {
      // return if shortcut key is detected from input, textarea
      if (
        event.target.tagName === 'INPUT' ||
        event.target.tagName === 'TEXTAREA'
      ) {
        return;
      }
      if (keys.some(key => event.key === key)) {
        // check if one of the key is part of the ones we want
        callbackRef.current(event);
      }
    },
    [keys],
  );
```

**Comment by arthurtyukayev** [comment]
@Nishant-Pall This functionality is already in the hook. And can be enabled using the "ignoreInputFields" option. It will do this exact thing. You can read more in the docs. 

What @thmi-inx was referring to was editable div elements, will require some additional modifications to the code base.

**Comment by Nishant-Pall** [comment]
@arthurtyukayev I see, I'll take a look into that, thanks for letting me know. This project has been of great help for me.
